releasetool/commands/tag/java.py

```diff
@@ -55,11 +55,21 @@ def kokoro_job_name(upstream_repo: str, package_name: str) -> Union[str, None]:
     if repo_short_name in java_framework_release_pool_repos:
         return f"cloud-java-frameworks/{repo_short_name}/stage"
 
+    if repo_short_name == "functions-framework-java" and package_name in functions_framework_java_packages:
+        if package_name == "java-function-invoker":
+            return f"functions-framework/java/invoker/release"
```

Comment:
nit: this seems a bit odd to have a special case for this single component vs. naming the job to match the component

Reply:
Changing the job name